downloads/1.2.x-release-notes/1.2.15.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 1.2.15 
+permalink: /downloads/1.2.x-release-notes/1.2.15/
+active: 1.2.15
+redirect_from: 
+    - /downloads/1.2.x-release-notes/
+---
+# Overview of jBallerina 1.2.15
```

Comment:
We seem to have used H3 for this in other release notes

Reply:
Fixed.